lib/web_ui/dev/test_runner.dart

```diff
@@ -124,6 +125,13 @@ class TestCommand extends Command<bool> with ArgUtils {
     // Check the flags to see what type of integration tests are requested.
     testTypesRequested = findTestType();
 
+    if (isSafariOnMacOS) {
+      /// Collect information on the bot.
+      final MacOSInfo macOsInfo = new MacOSInfo();
+      await macOsInfo.collectInformation();
+      return true;
```

Comment:
I can't tell if this is intentional or a remnant from debugging.

Reply:
This is intentional. Safari tests are still failing on some method version combinations. I'm adding this code to collect information on the bots. The return with true without running the tests.

Reply:
I added isLUCI check, so we can still use `felt test --browser=safari` locally :)